### PR TITLE
add python examples

### PR DESCRIPTION
**Pull request by ubbabeck**
# About pr

Added python examples to the docs. 

ref this issue https://github.com/breez/breez-sdk-docs/issues/8

### CODE REVIEW

File: src/guide/lnurl_pay.md [hunk 1]
```diff
@@ -59,6 +59,26 @@ try {
...
+<div slot="title">Python</div>
+<section>
+
+```python
+# Endpoint can also be of the form:
+# lnurlp://domain.com/lnurl-pay?key=val
+# lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttsv9un7um9wdekjmmw84jxywf5x43rvv35xgmr2enrxanr2cfcvsmnwe3jxcukvde48qukgdec89snwde3vfjxvepjxpjnjvtpxd3kvdnxx5crxwpjvyunsephsz36jf
+lnurl_pay_url = "lightning@address.com"
+ 
+parsed_input = breez_sdk.parse_input(lnurl_auth_url)
```
Comment (erdemyerebasmaz) [inline src/guide/lnurl_pay.md:73]:
```suggestion
parsed_input = breez_sdk.parse_input(lnurl_pay_url)
```

File: src/guide/lnurl_withdraw.md [hunk 2]
```diff
@@ -61,6 +61,23 @@ try {
...
+```python
+# Endpoint can also be of the form:
+# lnurlw://domain.com/lnurl-withdraw?key=val
+lnurl_withdraw_url = "lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4exctthd96xserjv9mn7um9wdekjmmw843xxwpexdnxzen9vgunsvfexq6rvdecx93rgdmyxcuxverrvcursenpxvukzv3c8qunsdecx33nzwpnvg6ryc3hv93nzvecxgcxgwp3h33lxk"
+
+
+parsed_input = breez_sdk.parse_input(lnurl_withdraw_url)
+if isinstance(parsed_input, breez_sdk.InputType.LN_URL_WITHDRAW)
+    amount_sats = parsed_input.data.min_withdrawable
+    result = sdk_service.withdraw_lnurl(parsed_input.data, amount_sats, "comment")
```
Comment (erdemyerebasmaz) [inline src/guide/lnurl_withdraw.md:76]: I haven't checked or python bindings yet, but there seems to be 2 different way we call API's: 
**breez_sdk** & **sdk_services**, what are the differences between them?

_(the latter seems to be a typo here when compared to majority)_
```suggestion
    result = sdk_services.withdraw_lnurl(parsed_input.data, amount_sats, "comment")
```

File: src/guide/receive_onchain.md [hunk 3]
```diff
@@ -143,4 +143,39 @@ try {
...
+```python
+refundables = sdk_services.list_refundables()
+```
+
+Once you have a refundable swap in hand, use the follwing code to execute a refund:
+
+```python
+destination_address = "..."
+sat_per_vbyte = <refund tx fee rate>
+sdk.refund(refundable.bitcoin_address, destination_address, sat_per_vbyte)
```
Comment (erdemyerebasmaz) [inline src/guide/receive_onchain.md:178]:
```suggestion
sdk_services.refund(refundable.bitcoin_address, destination_address, sat_per_vbyte)
```

File: src/guide/lnurl_withdraw.md (hunk 2 above)
Comment (ubbabeck) [reply src/guide/lnurl_withdraw.md:76]: The node functionality has to be called by the sdk_services by calling the `init_services`.  The `breez_sdk` is only the package in this case.

**Pull request merged**